monkey/infection_monkey/monkey.py

```diff
@@ -186,11 +186,11 @@ def start(self):
                     for exploiter in [exploiter(machine) for exploiter in self._exploiters]:
                         if self.try_exploiting(machine, exploiter):
                             host_exploited = True
-                            VictimHostTelem('T1210', ScanStatus.USED.value, machine=machine).send()
+                            VictimHostTelem('T1210', ScanStatus.USED, machine).send()
```

Comment:
I think that `ScanStatus.USED` is enum type object which will not serialize. Better keep `.value` which will make sure you send the corresponding int 1,2 or 3